### PR TITLE
Add remove invitations hook

### PR DESCRIPTION
**Pull request by Mariya-Sharibchenko**

### CODE REVIEW

**Review by Karador — APPROVED** [review]

File: src/core/hooks/InvitationsGraphQL/useRemoveInvitation.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { IInvitationDataProps } from 'context';
+import { useRemoveInvitationMutation } from 'core/graphql';
+import { allInvitationsQuery } from 'core/operations';
+
+export interface IRemoveInvitation {
+  invitations: IInvitationDataProps[],
+  companyId: string
+}
+
+export const useRemoveInvitation = (): [(props: IRemoveInvitation) => void, (invitations: IInvitationDataProps[]) => void] => {
```
Comment (suicaed) [inline src/core/hooks/InvitationsGraphQL/useRemoveInvitation.ts:12]: Move out return type.

Comment (suicaed) [reply src/core/hooks/InvitationsGraphQL/useRemoveInvitation.ts:12]: `(props: IRemoveInvitationProps): RemoveInvitationHookType`
For example.

**Review by 5u1c43d — CHANGES_REQUESTED** [review]

File: src/core/hooks/InvitationsGraphQL/useRemoveInvitation.ts (hunk 1 above)
Comment (Mariya-Sharibchenko) [reply src/core/hooks/InvitationsGraphQL/useRemoveInvitation.ts]: Pls, see commit below, is that what you mean?

**Review by 5u1c43d — APPROVED** [review]
Well done!

**Pull request merged**